src/components/styles/grandBackground.css

```diff
@@ -0,0 +1,4 @@
tXv39fXu21deVXLly68uvLr06/0uHZ8ELmhc0EEUUUVFFF9/VX1VUnXUnXVXX91111JK6c+fPlbavr6+vr6tq93lbz6y9eviyy9Z49+LDM6KEEEEKUEWWUVL9fX191V1p1VpV1V111V1VJKtPnz5222r39fX39e/dVVt68uvLnF3rryPXp4vjOhBChChFBFFFRRfX39/VVVdVaSadVdVVdf39VJOuvtOfLlbff19e/qqur2l7XLrVz8HxeU+bPN4UIUUUIIoosqOOOL6+vqquqtOutKuqqqqr6qrqVVXac+vW2+6vr39VW/Vtttt622+L4vi2+fXChBBBBBFBFFFFR/X191/dddVddSdVVdVf3V91fVtrr58uXW2+/q33Xb591W229ap0q71ZTZ4FNFFFNCKKKKijii+vqquuqqpOutOuqqr6q+vr6+qlXXy5deve/fv3fFttVt62+Pd829bVZ0k400UUUIIIooqKKKL6+q6qq6qqkq666qqqvv6+vuqpJc+fLly6+Pfj31Xe+99b/K6z368iiiihChBFFRFRfUf39VVVJVVV1JVVVVV/X1V9fdVVVqr5cuvi+PdvK+11tvXzfN6+fXk0I0oU0oo0qKKKL6+vv6qqqqrqrqqqqq91VfVX19VVWrny58uvi+Lar1vXxbfF63xel8W976I0UUIo0UoooqL6+vr6qqqTSqqSqSqqq+qqvr6q+vr6tVd5Vd7vW/VXW23+r18XxOnX+RTRTSgjSpYooovr6+vr6qrSrqrqSqSq9/VX19fXv39e6klVVVb1vm3rfd6XxbfN/xNFNNNKKlKKKi+vr6+vqqqqpKur6qrqt91fVX17+vr6+q6rVbyt62337tt9+JfF6+b083+RRRRTQgopYooovr6+vqq2qpKqqqqqr6qqq+rff179/VtqSVq6223xbVPF4/zelXTxP6PSzhTTSipYooooovr6qqqqqqqqq+qqqqvr37+vfv3791KqpXlbb30rb7tNXTr/F41eL4v8zjTTShSioopYvr6+vr6qqqqqtqqqq+rarfr39e/r3VVVV7tVVVtt622Xj1vmeFxq/xNFlNNKliKKKlii+vf19fVVtVVVVtVVVXv69+/cv19e/dVttvtcrbb9W979+b069aev8AibKaaaUUbFFFFF9e/qqq2qqqqq222r6qqqt9+/fv379223rVbbbfdVX39e7Z4t6/1P6NNNNNNllllii+vr6qq91VW221W21VW+7ffu337l9+/dv0ralUreS91Re5aj1SvtdPE/xlNnGymymyy+/fuX379+7baqqqrUqrbfdvuqr37l9+7barbbV1rvK/f1HF0fm9PE/yPTjw48enTxOn83xevXr16cuXm/xf6nXlbevXl15cuvX+J04+L/c/ieP/xAAmEQAABgMBAQADAQEAAwAAAAAAARARITEgMEBQYQJBYFFxEnCB/9oACAEDAQM/AOukJPqEQ+J8DWH0EZByRnQ9t+GaECMxQIWLWspQw/6BUGOkY9No3hEhVhJCk/aRs+h3FxrvwyBLHFYg9L43uridHEUCIMJbGNkI4tWMMt+UanxMRi8iViR1gxaX2UCFD4HFSLDJW0kL8RB6IzsQYtL6aFCEjlJIWdLFnfSWVcbIxCNNab1VtnEjSkfitI/6CYOn7ST12LSxfR9DEaEhgy4jVz4LFixfKQYG2iuOJK0tI2QaWl98HysQg0vS4vCxJixaTqjOQ+lpEclVsgQJxhb6SDgiD904yeEckFnQcMQcEYjjkx/g/W+8HFi9clukhANwxFzTexsC8BuYyIzBmYk8L8YzT4CFA+QjRiF43qvjnGsTBkDMkfmYvycgYsX5dCkPlk0sWsbLF9MiEY+cxAc/yFixfLPHQqOg0Z0vuY8Z0QXU34fkkpfgMJWcHbGeaGFi9E432yQIumw5mLwlY7XLGgQoMXSxTqvirdIrshbxjbfKxrHSxGHS8YxnnJ9BmRCK6iY5EnIkxaXjfBGqROMJBBi6SKVg9Fi/BIo6YBMYsXkxJe+NTZwlBmNkoVysGLdK3otaSvJjK8pyvK0nKtMpJD4CYgQguhjEmkreT99IxdNwHPyaDsjkXU5CK4p6I7DDl/0xBpYg8HPmrWY/SGThmBiOgoFCVvZeiUjGtUkJFLXRcpP5br03w0KBERiCH6LpNjj99sitMkkpJCkbqtJR+K8LWsq0SQvpIiMX2RtpIIUCPocP+JAt96o4yb04SUnZCUJ6YEWC/wDonO870TyQYkgYbnYg0pP5bI5K1zXwQQ+pHM34i8L6ZWtEYMYYullvynYQI52Iw5i9c6b0PjWmlgQXPH0O5i/Jha6I03qvfQoVpoUhmQrlYXKX2SsGlb65CJxAP/eFge4hSUlaZsORchMchisGZM4cXw2jbY3SDSkreREPok547S9J66zYMH4mH2CDnfHfXS0lJ/m8gRFjeD7rFi0ZJB8xCSSd0D/xI5Bmdj7ge+++Ub8uAiFz5FJS1hWJsQjMsztwZEZBwSv3EsoyVlWdI+bL/oJLFj9ELlLV98idkYvrrCQZMsECybBiB/kfNe2FgMR51jSSjAjIhBA4gMQ+j6Cw+oQYqBG6XgfDeuM2yrTBBjBkhNeDEcL9Um6IS+CMq1ujWCFIQo0MfQTbD5YznOkoVqYEhhv2yF+hAMGykD8JsHPVWUaHzNsbE915mfMZYTgaHg3V921vgFo+43y3og91ZkjY/ci772EKFf8AcKwpazpPusk+9E43onClrZG6Fv2yUtV9zlk5lylwn4MpXH97G8g0PzZ5z9ivfbib1IyoX4v3uP8AiD95yxlTwla8G+R8p/h5/hW7K82NMaa9SdTF7M6G9gtUCMH90sSNG/gCB63DEf8A6NMV/CO38MfgF4c+42qf4ef4ecJ92MJScJ9ad8CRPvNpn1n0OGMg61hXrNqYP6pLORltvfHi2INaS8aSUjCMIz//2Q=='/%3E%3C/defs%3E%3C/svg%3E");
```

Comment:
This is just doing `base64` on the actual SVG image, it doesn't <count> as rewriting it to CSS